lib/text_message.rb

```diff
@@ -0,0 +1,18 @@
+require 'twilio-ruby'
+
+class TextMessage
+  attr_reader :account_sid, :auth_token, :client
+
+  def initialize
+    @account_sid = ENV['accsid'] # Your Account SID from www.twilio.com/console
+    @auth_token = ENV['token']   # Your Auth Token from www.twilio.com/console
+    @client = Twilio::REST::Client.new account_sid, auth_token
```

Comment:
There's an opportunity for the Twilio client to be passed in as a constructor argument here, so that this class could be tested using mocking